oidc/transport.go

```diff
@@ -47,24 +47,24 @@ type AuthenticatedTransport struct {
 	http.RoundTripper
 
 	mu  sync.Mutex
```

Comment:
:( good point. But that doesn't help me - I need to set an initial JWT. Could add a getter and setter I guess, which seems the safest.